client/verta/verta/_registry/modelversions.py

```diff
@@ -64,3 +72,9 @@ def page_limit(self, msg):
 
     def page_number(self, msg):
         return msg.pagination.page_number
+
+    def with_workspace(self, workspace_name=None):
+        new_list = copy.deepcopy(self)
+        new_list._msg.id.registered_model_id = 0
```

Comment:
Like my comment above, I wonder if
```suggestion
        new_list._msg.id.ClearField('registered_model_id')
```
would work, for consistency with above?

Reply:
works too